versioned/storage/dynamodb/src/main/java/org/projectnessie/versioned/storage/dynamodb/DynamoClientProducer.java

```diff
@@ -18,7 +18,7 @@
 import java.net.URI;
 import org.immutables.value.Value;
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
-import software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient;
+import software.amazon.awssdk.http.apache.ApacheHttpClient;
```

Comment:
whats the reasoning for only doing this change here but not also in `DynamoDatabaseClient.java` ?

Reply:
That's code that'll go away.